project/panel.rkt

```diff
@@ -150,13 +153,26 @@ modifier author: Lîm Tsú-thuàn(GitHub: @dannypsnl)
     (define (remove-path-and-refresh btn event)
       (delete-directory/files (send view get-cur-selected-file) #:must-exist? #f)
       (send view reset-directory (send current-project get)))
+    (define (rename-path-and-refresh btn event)
+      (define name (get-text-from-user "new name for selected path?" ""))
```

Comment:
still missing `parent` and `init-val`

Reply:
1. `init-val` here: origin path
2. `parent` here: `#f`

Reply:
I am sorry. I didn't see all the message from the last requested changes.

Reply:
No worry, this is why we do reviewing